includes/Admin/Settings_Screens/Connection.php

```diff
@@ -159,16 +175,24 @@ public function render() {
 		);
 
 		// if the catalog ID is set, update the URL and try to get its name for display
-		if ( $catalog_id = $static_items['catalog']['value'] ) {
-
-			$static_items['catalog']['url'] = "https://facebook.com/products/catalogs/{$catalog_id}";
-
+		$catalog_id = $static_items['catalog']['value'];
+		$business_id = $static_items['business-manager']['value'];
+		if (!empty($catalog_id)) {
```

Comment:
Formatting, looks like everywhere  in the code there are spaces  
```suggestion
		if ( !empty( $catalog_id  ) ) {
```